application/clicommands/RescanCommand.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/* Icinga Web 2 X.509 Module | (c) 2022 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\X509\Clicommands;
+
+use Icinga\Application\Logger;
+use Icinga\Module\X509\CertificateUtils;
+
+class RescanCommand extends ScanCommand
+{
+    /**
+     * Rescans already known IP and port to find X.509 certificates.
+     *
+     * USAGE
+     *
+     * icingacli x509 rescan [options]
+     *
+     * OPTIONS:
+     *
+     *   --job=<name>          Rescan known IPs which belong to the given job.
+     *   --last-scan='2days'   Rescan targets whose last scan is older than the specified time
+     */
+    public function indexAction()
+    {
+        $lastScan = $this->params->shift('last-scan');
+        if ($lastScan) {
+            if (! ($value = strtotime($lastScan, 0))) {
```

Comment:
Also, I don't like assignments in `if`s. I would do it like this:

```
// You don't have to shift, if I'm not mistaken ...
$sinceLastScan = $this->params->get('since-last-scan');
if ($sinceLastScan !== null) {
    $sinceLastScan = strtotime($sinceLastScan);
    if ($sinceLastScan === false) {
        throw new InvalidArgumentException(... $this->params->get('since-last-scan') ...);
    } 
}